app/src/rgb_underglow.c

```diff
@@ -244,6 +246,11 @@ static int zmk_rgb_underglow_init(struct device *_arg) {
         return -EINVAL;
     }
 
+    ext_power = device_get_binding("EXT_POWER");
```

Comment:
You should change this to be wrapped in an `#if`, same with the actual declaration of the static variable.